torch/onnx/symbolic_opset12.py

```diff
@@ -71,3 +70,18 @@ def nll_loss(g, self, target, weight, reduction, ignore_index):
 
 def nll_loss2d(g, self, target, weight, reduction, ignore_index):
     return nll_loss(g, self, target, weight, reduction, ignore_index)
+
+def broadcast_tensors(g, tensor_list):
```

Comment:
could you leave a comment explaining what we are doing here and why we have a partial support of this op